addons/packages/prometheus/2.37.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+  - annotations:
+      kbld.carvel.dev/id: bitnami/kube-state-metrics:2.6.0
+      kbld.carvel.dev/origins: |
+        - resolved:
+            tag: 2.6.0
+            url: bitnami/kube-state-metrics:2.6.0
+    image: index.docker.io/bitnami/kube-state-metrics@sha256:8d5e0197a99ac85184ee37acfe8c85f705b1f4d619f5889b1bf489314499ad7a
+  - annotations:
+      kbld.carvel.dev/id: gcr.io/cadvisor/cadvisor:v0.44.0
+      kbld.carvel.dev/origins: |
+        - resolved:
+            tag: v0.44.0
+            url: gcr.io/cadvisor/cadvisor:v0.44.0
+    image: gcr.io/cadvisor/cadvisor@sha256:ef1e224267584fc9cb8d189867f178598443c122d9068686f9c3898c735b711f
```

Comment:
You can delete all resources related to cadvisor, because it is not actually used and was deleted in the last prometheus package.

Reply:
yea i remember reading the one in kubelet is what works but saw a few reference to cadvisor in the last package so thought we will need them. will remove them now.